<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Range</class>
 <widget class="QWidget" name="Range">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:transparent;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="m_pMainWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTreeView" name="treeView">
          <property name="styleSheet">
           <string notr="true">QTreeView {
   border: 0px;
}</string>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectItems</enum>
          </property>
          <attribute name="headerDefaultSectionSize">
           <number>40</number>
          </attribute>
          <attribute name="headerStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Apply to all columns</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="toolButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="list-add">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>Range</receiver>
   <slot>slotAddClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>Range</receiver>
   <slot>slotAllColumns(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotAddClicked()</slot>
  <slot>slotAllColumns(bool)</slot>
 </slots>
</ui>
